README.md

```diff
@@ -8,20 +8,41 @@ CoEpi Cloud API on AWS.
 * **Routing and Load Balancing**: API Gateway
 * **Permissions Management**: IAM
 
-
 ## Build
 
+JDK 11 or newer required
 
-```
+```sh
 gradle build
 ```
 
 Then, run:
 
-```
+```sh
 ./gradlew shadowJar
 ```
 
+## AWS Deploy via Terraform
+
+1. Ensure you have the AWS CLI configured, working, and pointing to the default AWS
+   account you wish to deploy to.
+2. Install Terraform 0.12 or newer
```

Comment:
How ? Any useful links ?

Reply:
General documentation links on Terraform should be useful

Reply:
The normal means of installing binaries - Homebrew for OSX, Choco for Win, package managers for *nix. Will add links.

Reply:
Done.